### ISSUE TITLE
the name `wr_translate_font_instance_platform_options` is defined multiple times

### ISSUE THREAD

**Issue by LovaTahina** [maps4print/azul#251]
## Description
   Compiling azul-text-layout v0.1.0 (https://github.com/maps4print/azul#7ed96030)
   Compiling azul-layout v0.1.0 (https://github.com/maps4print/azul#7ed96030)
   Compiling azul v0.1.0 (https://github.com/maps4print/azul#7ed96030)
error[E0428]: the name `wr_translate_font_instance_platform_options` is defined multiple times
   --> C:\Users\xyz\.cargo\git\checkouts\azul-f3b1e21a5e685cad\7ed9603\cargo\azul\../../azul\wr_translate.rs:646:1
    |
618 | const fn wr_translate_font_instance_platform_options(fio: FontInstancePlatformOptions) -> WrFontInstancePlatformOptions {
    | ----------------------------------------------------------------------------------------------------------------------- previous definition of the value `wr_translate_font_instance_platform_options` here
...
646 | fn wr_translate_font_instance_platform_options(fio: FontInstancePlatformOptions) -> WrFontInstancePlatformOptions {
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `wr_translate_font_instance_platform_options` redefined here
    |
    = note: `wr_translate_font_instance_platform_options` must be defined only once in the value namespace of this module

error[E0432]: unresolved import `crate::glutin::platform::unix`
    --> C:\Users\xyz\.cargo\git\checkouts\azul-f3b1e21a5e685cad\7ed9603\cargo\azul\../../azul\window.rs:1007:34
     |
1007 |     use crate::glutin::platform::unix::EventLoopWindowTargetExtUnix;
     |                                  ^^^^ could not find `unix` in `platform`

error[E0432]: unresolved imports `webrender::api::FontLCDFilter`, `webrender::api::FontHinting`
  --> C:\Users\xyz\.cargo\git\checkouts\azul-f3b1e21a5e685cad\7ed9603\cargo\azul\../../azul\wr_translate.rs:53:5
   |
53 |     FontLCDFilter as WrFontLCDFilter,
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `FontLCDFilter` in `api`
54 |     FontHinting as WrFontHinting,
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `FontHinting` in `api`

error[E0107]: wrong number of type arguments: expected 1, found 0
   --> C:\Users\xyz\.cargo\git\checkouts\azul-f3b1e21a5e685cad\7ed9603\cargo\azul\../../azul\wr_translate.rs:123:73
    |
123 |     pub(crate) fn translate_logical_position(input: LogicalPosition) -> WinitLogicalPosition {
    |                                                                         ^^^^^^^^^^^^^^^^^^^^ expected 1 type argument

error[E0107]: wrong number of type arguments: expected 1, found 0
   --> C:\Users\xyz\.cargo\git\checkouts\azul-f3b1e21a5e685cad\7ed9603\cargo\azul\../../azul\wr_translate.rs:128:65
    |
128 |     pub(crate) fn translate_logical_size(input: LogicalSize) -> WinitLogicalSize {
    |                                                                 ^^^^^^^^^^^^^^^^ expected 1 type argument

error[E0107]: wrong number of type arguments: expected 1, found 0
   --> C:\Users\xyz\.cargo\git\checkouts\azul-f3b1e21a5e685cad\7ed9603\cargo\azul\../../azul\wr_translate.rs:133:65
    |
133 |     pub(crate) const fn translate_winit_logical_position(input: WinitLogicalPosition) -> LogicalPosition {
    |                                                                 ^^^^^^^^^^^^^^^^^^^^ expected 1 type argument

error[E0107]: wrong number of type arguments: expected 1, found 0
   --> C:\Users\xyz\.cargo\git\checkouts\azul-f3b1e21a5e685cad\7ed9603\cargo\azul\../../azul\wr_translate.rs:138:61
    |
138 |     pub(crate) const fn translate_winit_logical_size(input: WinitLogicalSize) -> LogicalSize {
    |                                                             ^^^^^^^^^^^^^^^^ expected 1 type argument

error[E0107]: wrong number of type arguments: expected 1, found 0
   --> C:\Users\xyz\.cargo\git\checkouts\azul-f3b1e21a5e685cad\7ed9603\cargo\azul\../../azul\wr_translate.rs:163:75
    |
163 |     pub(crate) fn translate_physical_position(input: PhysicalPosition) -> WinitPhysicalPosition {
    |                                                                           ^^^^^^^^^^^^^^^^^^^^^ expected 1 type argument

error[E0107]: wrong number of type arguments: expected 1, found 0
   --> C:\Users\xyz\.cargo\git\checkouts\azul-f3b1e21a5e685cad\7ed9603\cargo\azul\../../azul\wr_translate.rs:167:67
    |
167 |     pub(crate) fn translate_physical_size(input: PhysicalSize) -> WinitPhysicalSize {
    |                                                                   ^^^^^^^^^^^^^^^^^ expected 1 type argument

error: aborting due to 9 previous errors

Some errors have detailed explanations: E0107, E0428, E0432.
For more information about an error, try `rustc --explain E0107`.
error: could not compile `azul`.

To learn more, run the command again with --verbose.
warning: build failed, waiting for other jobs to finish...
error: build failed

## Version / OS
Windows10

* azul version:
last version

* Operating system:
Windows

* Windowing system (X11 or Wayland, Linux only):

## Steps to Reproduce
At compilation

**Comment by fschutt** [comment]
Duplicate of #248
